department-of-veterans-affairs/caseflow
# APPEALS-54206, APPEALS-54168, APPEALS-53058: MST Legacy Issue Update Disposition Mismatch

[APPEALS-54206](https://jira.devops.va.gov/browse/APPEALS-54206) | IssueUpdate task Legacy Disposition Mismatch (MST)
[APPEALS-54168](https://jira.devops.va.gov/browse/APPEALS-54168) | IssueUpdate task Legacy Code Mismatch (MST)
[APPEALS-53058](https://jira.devops.va.gov/browse/APPEALS-53058) | Unreachable code in intake edit page (MST)

# Description
[APPEALS-54206](https://jira.devops.va.gov/browse/APPEALS-54206) | IssueUpdate task Legacy Disposition Mismatch (MST)
1. Removed disposition logging for legacy appeal issues on the IssueUpdateTask.
[APPEALS-54168](https://jira.devops.va.gov/browse/APPEALS-54168) | IssueUpdate task Legacy Code Mismatch (MST)
1. Updated Legacy Issue Update task to display Disposition Code instead of Level 1 Code

[APPEALS-53058](https://jira.devops.va.gov/browse/APPEALS-53058) | Unreachable code in intake edit page (MST)
1. Removed unreachable code that was introduced with MST/PACT release

## Acceptance Criteria
- [ ] Disposition displayed in IssueUpdateTask for legacy appeals matches the disposition of the issue updated.
- [ ] User can add/edit issues in Intake without any issues. 

## Testing Plan
[APPEALS-54206](https://jira.devops.va.gov/browse/APPEALS-54206) | IssueUpdate task Legacy Disposition Mismatch (MST)
1. Testing plan in ticket
[APPEALS-54168](https://jira.devops.va.gov/browse/APPEALS-54168) | IssueUpdate task Legacy Code Mismatch (MST)
1. Testing plan in ticket

[APPEALS-53058](https://jira.devops.va.gov/browse/APPEALS-53058) | Unreachable code in intake edit page (MST)
1. Log in as BVADWISE
2. Navigate to Intake
3. Select Board Appeal from the dropdown and click continue
4. Enter 400000004 for the veteran file number and click continue
5. Fill out form and click continue
6. Click **add issue**
7. Fill out the form and select **mst** checkbox
8. Click **establish appeal**
9. Click **correct the issues**
10. Click **select action** and select **edit issue**
11. uncheck MST from the modal and click save
12. Click **add issue**
13. Fill out the form to add a new issue. Click **add issue**
14. Click **save** and **confirm** on the modal
15. Expect success banner showing an issue was added and an issue was edited
![image](https://github.com/user-attachments/assets/d5458a9b-fa9a-4273-b564-e92e48f6296b)
